python/Ganga/Core/GangaRepository/SubJobXMLList.py

```diff
@@ -57,6 +57,8 @@ def __init__(self, jobDirectory='', registry=None, dataFileName='data', load_bac
             return
 
         self._subjobIndexData = {}
+        if parent is not None:
+            self._setParent(parent)
```

Comment:
The ```SubJobXMLList``` class doesn't always have it's parent set before it's fully constructed and used.
This class was written under the 'old-style' way of working so I'm planning to try and fix it up now that Core handles the ```_parent``` object and other functions in a much nicer/consistent way.